scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java

```diff
@@ -82,6 +82,15 @@ private void startCouchbase() {
             .password(password)
             .roles(Collections.singletonList(new UserRole("bucket_full_access", name))),
         true);
+
+    String configName = "configurations";
+    couchbase.createBucket(
+        DefaultBucketSettings.builder().name(configName).password(password).quota(100).build(),
```

Comment:
why did you use `quota(100)`?

Reply:
Minimally allowed bucket quota. 
Actually, this is a redundant code here since `DefaultBucketSettings` sets `100` as the default value.

Reply:
So can we remove such settings from this statement?